src/UI/examples/Table/Data/base.php

```diff
@@ -227,6 +231,7 @@ protected function getRecords(Range $range = null, Order $order = null): array
             ->data('a data table', $columns, $data_retrieval)
             ->withId('example_base')
             ->withActions($actions)
+            ->withNumberOfRows($current_user_pref_limit)
```

Comment:
Hello @nhaagen

The function "withNumberOfRows($current_user_pref_limit)" should not be documented additionally, because it should no longer be available with ILIAS 10. (See "Abandon Hits/Page as a User Setting" https://docu.ilias.de/goto_docu_wiki_wpage_8182_1357.html)

Can you confirm that the following thoughts are correct or correct them?
* Global default for all tables is the row quantity of25
* Developers can define their own default for a specific table (5 - ∞) (by parameterizing the table when using the UI Data Table)
* Users can customize the row specification for a specific table. The information is saved in the session, if implemented.

Remark:
The user preference is abolished for old tables (see "Abandon Hits/Page as a User Setting") 
As old tables still require information about the number of rows, the default value =25 is also transferred here. It is not possible for the user to adjust the default across the platform. The user can determine the number of rows for individual tables, which ILIAS remembers in the session or permanently.

Thanks and best regards
Oliver